package.json

```diff
@@ -14,6 +14,9 @@
     "cover:check": "istanbul check-coverage && echo code coverage thresholds met, achievement unlocked!",
     "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/init.js test/*-test.js"
   },
+  "pre-commit": [
+    "test"
```

Comment:
Ok, I'll kill the thing. :)

My intention was to automate things, to reduce human workload. :)